src/shogun/features/StringFeatures.cpp

```diff
@@ -78,19 +78,22 @@ template<class ST> CStringFeatures<ST>::CStringFeatures(CAlphabet* alpha)
 }
 
 template<class ST> CStringFeatures<ST>::CStringFeatures(const CStringFeatures & orig)
-: CFeatures(orig), num_vectors(orig.num_vectors),
-	single_string(orig.single_string),
-	length_of_single_string(orig.length_of_single_string),
-	max_string_length(orig.max_string_length),
-	num_symbols(orig.num_symbols),
-	original_num_symbols(orig.original_num_symbols),
-	order(orig.order), preprocess_on_get(false),
-	feature_cache(NULL)
+: CFeatures(orig)
 {
 	init();
 
 	ASSERT(orig.single_string == NULL) //not implemented
 
+	num_vectors = orig.num_vectors;
```

Comment:
General question. Does any of this affect the core PR changes? Because in general, I would prefer if we did those fixes in a different PR and keep this one strictly for adding the `get_features` of string features .... just to keep it a bit more tidy